docs/sources/reference/run.md

```diff
@@ -635,7 +635,51 @@ This can be overridden using a third `:rwm` set of options to each `--device` fl
 
 In addition to `--privileged`, the operator can have fine grain control over the
 capabilities using `--cap-add` and `--cap-drop`. By default, Docker has a default
-list of capabilities that are kept. Both flags support the value `all`, so if the
+list of capabilities that are kept. Here is a table to list the reference information on capabilities.
```

Comment:
Instead of "Here is a table.." can we make that sentence: "*The following table lists the Linux capability options which can be added or dropped:*"